edgecast/resources/cps/certificate.go

```diff
@@ -318,6 +332,11 @@ func setCertificateState(
 	flattenedDeployments := FlattenDeployments(resp.Deployments)
 	d.Set("deployments", flattenedDeployments)
 
+	if nresp != nil {
```

Comment:
Did you mean to use `sresp` to set the status?

Reply:
Issue with resolving merge conflicts. Thanks for catching it! Fixed.